packages/assistify-help-request/client/views/tabbar/HelpRequestActions.js

```diff
@@ -1,5 +1,10 @@
 import { RocketChat, UiTextContext } from 'meteor/rocketchat:lib';
 
+const showClosingComment = function() {
```

Comment:
The function was invoked to determine if the close comment section should be visible or not. I tried a lot to have this function referenced to sweet alert config using inline/shorthand declaration. However, did not work :(.